packages/jazz-tools/src/inspector/viewer/page.tsx

```diff
@@ -86,6 +86,16 @@ type PageProps = {
   className?: string;
 };
 
+function canEdit(value: RawCoValue) {
+  try {
+    const myRole = value.group.myRole();
+
+    return myRole === "writer" || myRole === "admin" || myRole === "writeOnly";
```

Comment:
Is it ok that people with writeOnly access are able to remove items from the CoList as well?

Reply:
Good point, they could actually, but it would be pretty messy.